contribs/gmf/less/mobile.less

```diff
@@ -145,32 +158,106 @@ span.twitter-typeahead {
     input {
       margin: 0;
       border: none;
-      width: ~"calc(" @search-width ~" - 2px)";
+      width: ~"calc(" @search-width ~" - 4.7em)";
+      left: 15px;
     }
     .clear-button {
       right: 0;
       left: auto;
     }
+    .form-control {
+      .noshadow();
```

Comment:
No need for a mixin here because:
 - we already use the "--autoprefix" option for lessc (See https://github.com/camptocamp/ngeo/blob/master/Makefile#L615). This is supposed to add the vendor prefixes when required.
 - we already have a `box-shadow` mixin we can use.